documentation/whatsnew/server/server-news-4.3.0.CR1.adoc

```diff
@@ -0,0 +1,21 @@
+= Server What's New in 4.3.0.CR1
+:page-layout: whatsnew
+:page-component_id: server
+:page-component_version: 4.3.0.CR1
+:page-feature_jbt_only: true
```

Comment:
it does not seem to be used anymore. I can't see any Awestruct extension using it.